src/sidebar/media-embedder.js

```diff
@@ -375,12 +376,17 @@ function replaceLinkWithEmbed(link) {
   // The link's text may or may not be percent encoded. The `link.href` property
   // will always be percent encoded. When comparing the two we need to be
   // agnostic as to which representation is used.
-  if (
-    link.href !== link.textContent &&
-    decodeURI(link.href) !== link.textContent
-  ) {
-    return null;
+  if (link.href !== link.textContent) {
```

Comment:
I feel like the preceding (pre-existing) comment could be clearer but when I try to re-write it myself, I get nowhere useful, so maybe it's fine...(feeling wishy-washy today!).